contrib/extractor.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# This is where the WoW client files are located
+CLIENT_DIR=/mnt/disk/client
```

Comment:
I think it's better to ask for the paths in this script

Reply:
Updated.